MovieApp/MovieApp/ViewController/ViewController.swift

```diff
@@ -0,0 +1,122 @@
+//
+//  ViewController.swift
+//  MovieApp
+//
+//  Created by Julio Gabriel Tobares on 12/04/2022.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+
+    @IBOutlet var NowMovies: UICollectionView!
```

Comment:
Use camel case for variables including outlets

Reply:
Names fixed